thavalon-webapp/src/components/gameComponents/playerBoard.tsx

```diff
@@ -140,6 +145,7 @@ export function PlayerBoard(): JSX.Element {
                     setVotes(voteMap);
                 } else {
                     // TODO: Trigger Maive Toast here.
```

Comment:
this TODO still needed?

Reply:
Removed since we're going with server-side toasts